### PR TITLE
Docking facelift

### PR DESCRIPTION
**Pull request by Limuzhou**

### CODE REVIEW

**Comment by Ramjii** [comment]
It's beautiful

File: nanome_docking/Docking.py [hunk 1]
```diff
@@ -29,15 +31,15 @@ def on_run(self):
         menu = self._menu
         # If menu doesn't have Receptor and Ligands selected, open it
         # Else, just start docking
-        if menu.is_ready_for_docking() == False:
-            self.open_menu()
-        else:
-            self.run_docking(menu.get_receptor(), menu.get_ligands(), menu.get_site(), menu.get_params())
+        # if menu.is_ready_for_docking() == False:
```
Comment (Astrovicis) [inline nanome_docking/Docking.py:34]: I would just delete these commented out lines.

File: nanome_docking/Docking.py [hunk 2]
```diff
@@ -101,6 +111,7 @@ def run_docking(self, receptor, ligands, site, params):
         if self._menu._run_button.unusable == True:
             return
         self._menu.make_plugin_usable(False)
+        #self._menu.show_loading(True)
```
Comment (Astrovicis) [inline nanome_docking/Docking.py:114]: same here

File: nanome_docking/Docking.py [hunk 3]
```diff
@@ -110,6 +121,7 @@ def run_docking(self, receptor, ligands, site, params):
 
         setup_structures = functools.partial(self.set_and_convert_structures, site != None, params)
         self.request_complexes(request_list, setup_structures)
+        #self._menu.show_loading(False)
```
Comment (Astrovicis) [inline nanome_docking/Docking.py:124]: and here

File: nanome_docking/_DockingMenu.py [hunk 4]
```diff
@@ -282,33 +480,48 @@ def visual_scores_button_pressed_callback(button):
         child.add_new_label()
 
         # loading menus
-        menu = nanome.ui.Menu.io.from_json(os.path.join(os.path.dirname(__file__), '_docking_menu.json'))
+        menu = nanome.ui.Menu.io.from_json(os.path.join(os.path.dirname(__file__), '_docking_menu_new.json'))
         self._plugin.menu = menu
+        self.setting_menu = nanome.ui.Menu.io.from_json(os.path.join(os.path.dirname(__file__), '_docking_setting_new.json'))
+        # self.setting_menu.index = 1
```
Comment (Astrovicis) [inline nanome_docking/_DockingMenu.py:486]: here too

**Pull request merged**